test/tools/fuzzing/README.md

```diff
@@ -17,4 +19,28 @@ python fuzz.py --afl_dir=/path/to/AFL -t timeout
 The default timeout is 60 minutes per test, It is not recommended to do much less than that.
 Using a timeout of 0 will cause the tests to run to completion or until you cancel the process with C-c (or any keyboard interrupt)
 
-Once a failing test is fixed, add it to the appropriate model directory to ensure it gets tested going forward.
+Once a failing test is fixed, add the input model file to the appropriate folder under the `models/` directory to ensure it gets tested going forward.
+
+## Interpreting Output
+
+The results are output into the `output/` directory, which will have the same top level structure as the `models/` directory. Each of these directories contain data from runs based on mutations of the corresponding input directories.
+
+The structure of each directory is as follows
+```
++ crashes
```

Comment:
What is the meaning behind the `+` and `-` here?

Reply:
its the most standard directory structure formatting I know of. `+` indicates directories, `-` indicates files